test/Utils/src/org/openj9/test/util/IncludeExcludeTestAnnotationTransformer.java

```diff
@@ -42,7 +42,7 @@
 		String line = null;
 		String excludeFile = System.getenv("EXCLUDE_FILE");
 		if (null == excludeFile) {
-			excludeFile = IncludeExcludeTestAnnotationTransformer.class.getClassLoader().getResource("excludes/default_exclude.txt").getFile();
+			excludeFile = IncludeExcludeTestAnnotationTransformer.class.getClassLoader().getResource("excludes/openj9_exclude.txt").getFile();
 		}
```

Comment:
I think we have to keep the default to be "default_exclude.txt".

Reply:
change default back to default_exclude.txt